implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs

```diff
@@ -288,7 +284,36 @@ impl NodeManagerWorker {
             Ok(Response::not_found(req.id()).body(InletStatus::new(
                 "".to_string(),
                 "".to_string(),
-                alias.clone(),
+                alias,
+                Some(format!("Inlet with alias {alias} not found").into()),
+                "".to_string(),
+            )))
+        }
+    }
+
+    pub(super) async fn show_inlet<'a>(
+        &mut self,
+        req: &Request<'_>,
+        alias: &'a str,
+    ) -> Result<ResponseBuilder<InletStatus<'a>>> {
+        let node_manager = self.node_manager.write().await;
```

Comment:
Same here: `self.node_manager.read().await;`

Reply:
Added in latest commit